kubetest2-gke/deployer/up.go

```diff
@@ -33,6 +35,17 @@ import (
 	"sigs.k8s.io/kubetest2/pkg/metadata"
 )
 
+// If one of the error patterns below is matched, it would be recommended to
+// retry creating the cluster in a different region.
+// - stockout
+// - nodes fail to start
+// - component is unhealthy
+var retryableCreationErrors = []*regexp.Regexp{
+	regexp.MustCompile(`.*does not have enough resources available to fulfill.*`),
```

Comment:
we should differentiate stockout errors from regular quota errors.

same concerns as: https://github.com/kubernetes-sigs/kubetest2/pull/13#issuecomment-668292386

Reply:
Ditto. The same reply as https://github.com/kubernetes-sigs/kubetest2/pull/85#discussion_r564119419